testsupport/tiers/checks.go

```diff
@@ -440,10 +440,10 @@ type appstudioTierChecks struct {
 
 func (a *appstudioTierChecks) GetNamespaceObjectChecks(_ string) []namespaceObjectsCheck {
 	checks := []namespaceObjectsCheck{
-		resourceQuotaComputeDeploy("20", "32Gi", "1750m", "32Gi"),
-		resourceQuotaComputeBuild("120", "128Gi", "12", "64Gi"),
+		resourceQuotaComputeDeploy("0", "32Gi", "1750m", "32Gi"),
+		resourceQuotaComputeBuild("0", "128Gi", "12", "64Gi"),
 		resourceQuotaStorage("50Gi", "200Gi", "50Gi", "30"),
-		limitRange("2", "2Gi", "10m", "256Mi"),
+		limitRange("0", "2Gi", "10m", "256Mi"),
```

Comment:
Should we put 200m instead of 10m as default cpu request as performance team tested?

Reply:
Yes, thanks.